lib/hanami/router/prefix.rb

```diff
@@ -10,6 +10,7 @@ class Prefix
       # @since 2.0.0
       # @api private
       def initialize(prefix)
+        prefix = DEFAULT_SEPARATOR if prefix.empty?
```

Comment:
If Prefix gets initialized as empty string, it was causing a test failure in the main framework suite.

Reply:
I notice that it is actually being used with empty string intentionally here:

https://github.com/hanami/router/blob/main/lib/hanami/router.rb#L78

So this is probably the wrong approach. Perhaps `UrlHelpers` should do this?